js/index.js

```diff
@@ -8,35 +8,100 @@ const siteContent = {
+    "product-h4": "Product",
+    "product-content":
+      "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4": "Vision",
+    "vision-content":
+      "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  contact: {
+    "contact-h4": "Contact",
+    address: "123 Way 456 Street Somewhere, USA",
+    phone: "1 (888) 888-8888",
+    email: "sales@greatidea.io"
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.src = siteContent["nav"]["img-src"];
+logo.alt = "Great Idea! Company logo.";
+
+const navItems = document.querySelectorAll("nav a");
+navItems.forEach((item, i) => {
+  item.textContent = siteContent.nav[`nav-item-${i + 1}`];
+});
+
+const header = document.getElementsByClassName("cta-text")[0];
+header.getElementsByTagName("h1")[0].innerHTML = siteContent["cta"]["h1"];
+header.getElementsByTagName("button")[0].innerHTML =
+  siteContent["cta"]["button"];
+const headerImg = document.getElementById("cta-img");
+headerImg.src = siteContent["cta"]["img-src"];
+
+const main = document.querySelectorAll(".text-content");
+main[0].getElementsByTagName("h4")[0].innerHTML =
+  siteContent["main-content"]["features-h4"];
+main[0].getElementsByTagName("p")[0].innerHTML =
+  siteContent["main-content"]["features-content"];
+main[1].getElementsByTagName("h4")[0].innerHTML =
+  siteContent["main-content"]["about-h4"];
+main[1].getElementsByTagName("p")[0].innerHTML =
+  siteContent["main-content"]["about-content"];
+main[2].getElementsByTagName("h4")[0].innerHTML =
+  siteContent["main-content"]["services-h4"];
+main[2].getElementsByTagName("p")[0].innerHTML =
+  siteContent["main-content"]["services-content"];
+main[3].getElementsByTagName("h4")[0].innerHTML =
+  siteContent["main-content"]["product-h4"];
+main[3].getElementsByTagName("p")[0].innerHTML =
+  siteContent["main-content"]["product-content"];
+main[4].getElementsByTagName("h4")[0].innerHTML =
+  siteContent["main-content"]["vision-h4"];
+main[4].getElementsByTagName("p")[0].innerHTML =
+  siteContent["main-content"]["vision-content"];
+
```

Comment:
this works and is correct but theres a lot of repeat, think of other ways to approach this problem, with a switch statment we can set the value of the string at the end of our statment to dynamically change it
```
for(let i = 0; i< mainContent.length; i++){
  let section = '';
  switch(i){
    case 0: section ='features'
    case 1: section = 'about'
    case 2: section = 'services'
    case 3: section = 'product'
    case 4: section = 'vision'
  }
main[i].getElementsByTagName("h4")[0].innerHTML = siteContent["main-content"][`${section}-h4`];
main[i].getElementsByTagName("p")[0].innerHTML = siteContent["main-content"]['${section}-content'];
}